exporter/datadogexporter/logs_exporter.go

```diff
@@ -0,0 +1,102 @@
+// Copyright  The OpenTelemetry Authors
```

Comment:
```suggestion
// Copyright The OpenTelemetry Authors
```
There is an extra space here in all the copyright headers.